setup.py

```diff
@@ -32,20 +32,29 @@
 
 import os
 import pip
+import sys
 from pip.req import parse_requirements
 from setuptools import setup, find_packages
 
 
 requirements = [str(requirement.req) for requirement in parse_requirements(
     'requirements.txt', session=pip.download.PipSession())]
 
+# Python Version check
+if not sys.version_info[0] == 2:
+    sys.exit('Python 3 is not supported')
```

Comment:
I tested it doesn`t work in Python3.

http://stackoverflow.com/questions/25445439/what-does-syntaxerror-missing-parentheses-in-call-to-print-mean-in-python